test/fixtures/messages.yml

```diff
@@ -23,6 +25,7 @@ identify_photo:
   content_text: What do you see in this photo?
   content_document: # the cat photo is not attached in this field, currently it's attached at the document level. info: https://platform.openai.com/docs/assistants/tools/knowledge-retrieval and https://platform.openai.com/docs/guides/vision
   run:
+  created_at: 2023-12-30 1:02:00
```

Comment:
Instead of hardcoding these, which introduces a maintenance burden on these files, what do you think of declaring a T value at the top of the file and then incrementing it by one minute for each of these? Like this:

```yml
<% t = DateTime.now %>

identify_photo:
  created_at: <% t += 1.minute %>
```

Or, if it's easy to do, declaring a helper function somewhere that's available in the yaml files?

Reply:
That's a good idea when we need the timestamp to be something specific. In this case, having it be in the past at some fixed point totally works. This is just populating "old" / existing conversations for a user so we shouldn't ever need to update these timestamps.

In this PR my update was because I had left the time off. I believe fixtures do not guarantee the order that it will insert records, so I was getting a flaky test when I was expecting the `.messages.first` to be consistent and it wasn't. Now we order messages by created_at so this ensures they'll be consistent in our tests.